### PR TITLE
bump tinygo version

### PR DESCRIPTION
**Pull request by dgryski**

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
           go-version: stable
```
Comment (cee-dub) [inline .github/workflows/validate.yml:42]: We talked about using `oldstable` here to keep from having breakage with new Go releases. Thoughts on bumping it back to that?

File: .github/workflows/validate.yml [hunk 2]
```diff
@@ -57,7 +57,4 @@ jobs:
         run: go test -v -race -tags="fastlyinternaldebug nofastlyhostcalls" ./...
 
       - name: Tests - TinyGo
-        env:
-          GOARCH: wasm
-          GOOS: wasip1
-        run: tinygo test -v -tags="fastlyinternaldebug nofastlyhostcalls" ./...
+        run: tinygo test -v -target=wasip1 -tags="fastlyinternaldebug nofastlyhostcalls" ./...
```
Comment (cee-dub) [inline .github/workflows/validate.yml:60]: Personally, I'd love to reduce noise in the test output by dropping `-v` here and elsewhere. Maybe a different PR?

**Review by cee-dub — APPROVED** [review]
LGTM. A couple optional things.

File: .github/workflows/validate.yml (hunk 2 above)
Comment (dgryski) [reply .github/workflows/validate.yml]: Done.

File: .github/workflows/validate.yml (hunk 1 above)
Comment (dgryski) [reply .github/workflows/validate.yml:42]: So the assumption is devs will discover breakages on their machines well before CI hits.  Ok.

**Comment by cee-dub** [comment]
Great!

**Review by cee-dub — APPROVED** [review]

**Pull request merged**